assets/js/app.js

```diff
@@ -16,9 +16,6 @@ import "phoenix_html";
 // Local files can be imported directly using relative paths, for example:
 // import socket from "./socket"
 
-// Import non-Lux Vue components
-import "./components/_components";
```

Comment:
This should not be deleted - once it is restored I can open this as a PR for review.

Reply:
I restored those two lines.